src/ctoken.ts

```diff
@@ -157,6 +159,17 @@ export function handleBorrow(event: Borrow): void {
     cTokenStats.totalUnderlyingBorrowed.plus(borrowAmountBD)
   cTokenStats.save()
 
+  if (event.params.accountBorrows.equals(event.params.borrowAmount)) {
+    // If this is the first borrow, increment the market's borrower count
```

Comment:
Determine whether this is the first borrow if the resulting `accountBorrows` param is equal to the `borrowAmount`. This should only increment if the borrower has no existing borrows in the market.